zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
```

Comment:
Are you sure about this version?

Reply:
Not at all. I simply copied the lines from other Zephyr CMakeLists.txt files and followed their usages.  What should it be?

Reply:
Updated to 3.13.1 to match the rest of Zephyr.